aws/ec2_test.go

```diff
@@ -1,49 +1,66 @@
 package aws
 
 import (
-	"bytes"
-	"math/rand"
 	"testing"
-	"time"
 
 	awsgo "github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/aws-nuke/util"
 	"github.com/gruntwork-io/gruntwork-cli/errors"
 	"github.com/stretchr/testify/assert"
 )
 
-// Returns a unique (ish) id we can attach to resources and tfstate files so they don't conflict with each other
-// Uses base 62 to generate a 6 character string that's unlikely to collide with the handful of tests we run in
-// parallel. Based on code here: http://stackoverflow.com/a/9543797/483528
-func uniqueID() string {
-
-	const BASE_62_CHARS = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
-	const UNIQUE_ID_LENGTH = 6 // Should be good for 62^6 = 56+ billion combinations
-
-	var out bytes.Buffer
-
-	rand := rand.New(rand.NewSource(time.Now().UnixNano()))
-	for i := 0; i < UNIQUE_ID_LENGTH; i++ {
-		out.WriteByte(BASE_62_CHARS[rand.Intn(len(BASE_62_CHARS))])
+func getLinuxAmiIDForRegion(region string) string {
```

Comment:
This will be a pain to maintain as regions are added or AMIs removed. You should be able to find the AMI automatically using Amazon's APIs, similar to Packer's [source_ami_filter](https://www.packer.io/docs/builders/amazon-ebs.html#source_ami_filter).